compiler/control/OMROptions.cpp

```diff
@@ -789,6 +789,8 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
                                           SET_OPTION_BIT(TR_EnableJitSamplingUpgradesDuringHWProfiling), "F", NOT_IN_SUBSET},
    {"enableUpgradingAllColdCompilations", "O\ttry to upgrade to warm all cold compilations", SET_OPTION_BIT(TR_EnableUpgradingAllColdCompilations), "F"},
    {"enableValueTracing",                 "O\tenable runtime value tracing (experimental)", SET_OPTION_BIT(TR_EnableValueTracing), "F"},
+   {"enableVectorAPIExpansion",          "M\tForce enable expansion of Vector API",               SET_OPTION_BIT(TR_EnableVectorAPIExpansion), "F"},
```

Comment:
Please add that `disableVectorAPIExpnasion` prevails.

Reply:
Sorry for being picky but I think it's better to remove "Force" from it. I think it might cause confusion with `forceVectorAPIExpansion` option.